cli/src/index.ts

```diff
@@ -26,4 +28,23 @@ program.addCommand(deleteCommand)
 program.addCommand(listFilesCommand)
 program.addCommand(showFileCommand)
 
-program.parse(process.argv)
+interface OptionValues {
+  config: string
+  node_url: string
+  wallet: string
+  password: string
+}
+
+const commandOptions: OptionValues = program.opts() as OptionValues
+
+if (existsSync(commandOptions.config)) {
+  const configOptions = parseConfigFile(commandOptions.config)
+
+  // commandOptions get priority over configOptions
+  const programOptions = setProgramOptions(commandOptions, configOptions)
+  for (const [key, value] of Object.entries(programOptions)) {
+    program.setOptionValue(key, value)
+  }
+}
```

Comment:
I feel like that should be in another file and/or in a function 🤔

Reply:
Why not! i'll look into implementing something see if it's cleaner

Reply:
implementing a function that takes program as a param is not optimal as program isn't genericly typed... i think despite gaining in ledgibility we might lose in cleaness